community/sway/etc/skel/.profile

```diff
@@ -15,9 +15,9 @@ export QT_WAYLAND_DISABLE_WINDOWDECORATION="1"
 
 # set default shell and terminal
 export SHELL=/usr/bin/zsh
-export TERM=xterm-kitty
-export TERMINAL_COMMAND='/usr/share/sway/scripts/terminal.sh'
+export TERM=xterm-256color
```

Comment:
```suggestion
```

Reply:
Yep, isn't working anyway. What's about changing $term to `foot --term xterm-256color`, are we doing it by default or we're leaving solving ssh issues to the users?

Reply:
didn't try it, but shouldn't adding 
```
alias ssh="TERM=xterm-256color ssh"
```
to the .zshrc be working? if so I'd prefer that

Reply:
That's a valid point (actually that's how I do it on my machine), but I fear that it can break things if user will install and use other terminal by default. It seems like a good solution, but we should somehow inform users that we made this. At this point it will lead to a giant readme, time to make a wiki? 😆

Reply:
If a user changes shells, I expect him to know what he's doing... Regarding wiki: we could actually use github wiki or github docs for that in the manjaro-sway repo. Perhaps not a full fledged wiki, but a 'faq' page, perhaps searchable..

Reply:
I mean: if a user breaks it, he will google and find a solution in no time.

Reply:
Makes sense to me. Then adding the mentioned line to .zshrc is a good middleground. Would you like to add it as a suggestion? I can do it myself, but, you know, I just kinda want to show that this PR was not made by me only 😄